schema/publisher.prop.jsd

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://chharvey.github.io/schemaorg-jsd/meta/member.jsd#",
+  "$id": "https://chharvey.github.io/schemaorg-jsd/schema/publisher.prop.jsd",
+  "title": "https://schema.org/publisher",
```

Comment:
Schema.org titles should be the canonical urls (`http`). This was fixed in eb145ae .